feat: max-overflow on Avatar.Group to be triggered by click or focus …

RiteshMakan opened the pull request:
…(#31924)

Based on this issue, https://github.com/ant-design/ant-design/issues/31924. Here is a proposal to allow 'click' or 'hover' to trigger the popover instead

<!--
First of all, thank you for your contribution! 😄

New feature please send a pull request to feature branch, and rest to master branch.
Pull requests will be merged after one of the collaborators approve.
Please makes sure that these forms are filled before submitting your pull request, thank you!
-->

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

<!--
1. Describe the source of requirement, like related issue link.
-->

### 💡 Background and solution

<!--
1. Describe the problem and the scenario.
2. GIF or snapshot should be provided if includes UI/interactive modification.
3. How to fix the problem, and list final API implementation and usage sample if that is a new feature.
-->

### 📝 Changelog

<!--
Describe changes from the user side, and list all potential break changes or other risks.
--->

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

RiteshMakan committed:
feat: max-overflow on Avatar.Group to be triggered by click or focus (#31924)

Based on this issue, https://github.com/ant-design/ant-design/issues/31924. Here is a proposal to allow 'click' or 'hover' to trigger the popover instead

Commit:
docs: added maxPopoverTrigger to API

Commit:
chore: updated demo

Commit:
test: updated snapshot

RiteshMakan committed:
Update components/avatar/index.en-US.md

Co-authored-by: xrkffgg <xrkffgg@vip.qq.com>

RiteshMakan committed:
Update components/avatar/index.zh-CN.md

Co-authored-by: xrkffgg <xrkffgg@vip.qq.com>

RiteshMakan commented:
Hi, all checks have passed, where can I ask for an approval to be able to merge this in?

xrkffgg commented:
> * 🚨 Please fill changelog in the PR
>   
>   * Write with a **developer-oriented perspective** and **narrative method**, without describing the details of the repair
>   * **Describing the problem and the impact on the developer**
>   * **describing the user-first site problem**, not your solution
>   * Refer: https://ant.design/changelog#4.9.0
> * 🚨 请填写 PR 中的 changelog
>   
>   * 请用**面向开发者的角度**和**叙述方式撰写**，不描述修复细节
>   * **描述问题和对开发者的影响**
>   * **描述用户第一现场的问题**，而非你的解决方式
>   * 参考：https://ant.design/changelog-cn#4.9.0

ref

xrkffgg committed:
ci: update pr check

dependabot[bot] committed:
chore(deps-dev): bump node-fetch from 2.6.1 to 3.0.0 (#32000)

Bumps [node-fetch](https://github.com/node-fetch/node-fetch) from 2.6.1 to 3.0.0.
- [Release notes](https://github.com/node-fetch/node-fetch/releases)
- [Changelog](https://github.com/node-fetch/node-fetch/blob/main/docs/CHANGELOG.md)
- [Commits](https://github.com/node-fetch/node-fetch/compare/v2.6.1...v3.0.0)

---
updated-dependencies:
- dependency-name: node-fetch
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

yingxirz committed:
Update buttons.zh-CN.md (#32012)

Commit:
Updated changelog

RiteshMakan commented:
> > * 🚨 Please fill changelog in the PR
> >   
> >   * Write with a **developer-oriented perspective** and **narrative method**, without describing the details of the repair
> >   * **Describing the problem and the impact on the developer**
> >   * **describing the user-first site problem**, not your solution
> >   * Refer: https://ant.design/changelog#4.9.0
> > * 🚨 请填写 PR 中的 changelog
> >   
> >   * 请用**面向开发者的角度**和**叙述方式撰写**，不描述修复细节
> >   * **描述问题和对开发者的影响**
> >   * **描述用户第一现场的问题**，而非你的解决方式
> >   * 参考：https://ant.design/changelog-cn#4.9.0
> 
> ref

Hi, I pushed an update to the Changelog. I hope my Google translate to Chinese is ok. I see that the bot has closed my issue

xrkffgg commented on CHANGELOG.en-US.md:
```diff
@@ -54,7 +54,9 @@ timeline: true
 - 💄 Add `@checkbox-border-radius` less variable. [#31360](https://github.com/ant-design/ant-design/pull/31360) [@Gherciu](https://github.com/Gherciu)
-- 🐞 Avatar add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
+- 🐞 Avatar
+  - add `crossOrigin` property to resolve cross origin issue. [#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
```
Keep it as it is here.

RiteshMakan replied:
👍 Thank you. I want to take the time to thank you for assisting me with this PR. I really appreciate it

RiteshMakan replied:
> Please input PR changelog

I have no idea what to do, I would like to learn. This is my first PR to ant-design

xrkffgg commented on CHANGELOG.zh-CN.md:
```diff
@@ -54,7 +54,9 @@ timeline: true
   - 🐞 修复 Tree 拖拽到外部时没有清空指示器的问题。[#478](https://github.com/react-component/tree/pull/478)
 - 💄 新增 Less 变量 `@checkbox-border-radius`。[#31360](https://github.com/ant-design/ant-design/pull/31360) [@Gherciu](https://github.com/Gherciu)
-- 🐞 Avatar 增加 `crossOrigin` 参数以解决跨域问题。[#31273](https://github.com/ant-design/ant-design/pull/31273) [@Map1en](https://github.com/Map1en)
+- 🐞 Avatar
```
Keep it as it is here.

xrkffgg commented:
Please input PR changelog

RiteshMakan commented:
> Please input PR changelog

I have no idea what to do, I would like to learn. This is my first PR to ant-design

xrkffgg commented:
![image](https://user-images.githubusercontent.com/29775873/131939265-0ebf7557-47bb-46c5-902d-c1a72fd82556.png)

open this

acfasj committed:
fix(date-picker): DatePicker ref typing (#31993)

Chtau committed:
fix: Added German translation for Image 'Preview' text (#32001)

zombieJ committed:
docs: Update Dropdown onVisibleChange desc
close #32024

RiteshMakan commented:
![Screenshot 2021-09-03 at 10 32 06](https://user-images.githubusercontent.com/82932397/131975780-53d9fb56-2ec0-4307-a0c3-ce462bc3078d.png)
Done

RiteshMakan committed:
Merge branch 'ant-design:master' into patch-1

xrkffgg commented:
There is some confusion, please open a new PR
